# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1264009749 18000
#      Wed Jan 20 12:49:09 2010 -0500
# Node ID 722b43fd486e9c9e1335d31073c0f4a64aa2fa65
# Parent  deba43069023e6d13282ff584722dea20c436930
don't distribute libcruft/STOP.patch

diff --git a/libcruft/ChangeLog b/libcruft/ChangeLog
--- a/libcruft/ChangeLog
+++ b/libcruft/ChangeLog
@@ -1,8 +1,12 @@
+2010-01-20  John W. Eaton  <jwe@octave.org>
+
+	* Makefile.am (EXTRA_DIST): Remove STOP.patch from the list.
+
 2010-01-20  Jaroslav Hajek  <highegg@gmail.com>
 
 	* misc/quit.h: Define inline-function variant of OCTAVE_QUIT in C++
 	mode.
 
 2009-01-13  Rik <octave@nomad.inbox5.com>
 
 	* STOP.patch: Remove unmaintained patch for Fortran code
diff --git a/libcruft/Makefile.am b/libcruft/Makefile.am
--- a/libcruft/Makefile.am
+++ b/libcruft/Makefile.am
@@ -58,12 +58,12 @@ include slatec-err/module.mk
 include slatec-fn/module.mk
 include villad/module.mk
 
 cruft.def: $(libcruft_la_SOURCES) mkf77def
 	chmod a+rx mkf77def
 	./mkf77def $(srcdir) $(libcruft_la_SOURCES) > $@-t
 	mv $@-t $@
 
-EXTRA_DIST += ChangeLog STOP.patch mkf77def.in
+EXTRA_DIST += ChangeLog mkf77def.in
 
 DISTCLEANFILES = cruft.def
 
